src/matchmaker.ts

```diff
@@ -199,17 +199,20 @@ export default class Matchmaker {
                         provider.removeListener('block', waitForTx)
                         const signedTx = Transaction.from(tx).serialized
                         console.log(`Found transaction hash: ${ firstTx.hash } onchain at block number: ${ tx.blockNumber }`)
-                        // TODO: Add params.inclusion.block target to mev_simBundle, not currently implemented in API
+                        if (!tx.blockNumber) {
```

Comment:
When would there not be a blockNumber? if it was pending?

Reply:
not sure, but the field is possibly null so it needs to be checked before use